LICENSE

```diff
@@ -13,7 +13,7 @@
       the copyright owner that is granting the License.
 
       "Legal Entity" shall mean the union of the acting entity and all
-      other entities that control, are controlled by, or are under common
+      other workspaceEntityConfig that control, are controlled by, or are under common
```

Comment:
Looks like you did a rename that went a little wide! :) Would you be able to revert this change?